site/docs/fargate-tutorials/django.md

```diff
@@ -38,34 +39,20 @@ Make sure to copy the PostgreSQL connection string as you will not see it again.
 2. The first migration calls the `iasql_install` SQL function to install the ECS simplified [module](../concepts/module.md) into the hosted database.
 
 ```sql title="psql postgres://d0va6ywg:nfdDh#EP4CyzveFr@db.iasql.com/_4b2bb09a59a411e4 -c"
-SELECT * from iasql_install(
-   'aws_ecs_simplified'
-);
+SELECT
+  *
+FROM
+  iasql_install ('aws_ecs_simplified');
 ```
 
 If the function call is successful, it will return a virtual table with a record for each new table in your database under `created_table_name` and the number of existing resources or records imported from the account under `record_count`.
 
 ```sql
-       module_name        |      created_table_name       | record_count
+module_name | created_table_name | record_count
 --------------------------+-------------------------------+--------------
- aws_cloudwatch           | log_group                     |            0
- aws_ecr                  | public_repository             |            0
- aws_ecr                  | repository                    |            1
- aws_ecr                  | repository_policy             |            0
- aws_security_group       | security_group                |            2
- aws_security_group       | security_group_rule           |            0
- aws_vpc                  | vpc                           |            1
- aws_vpc                  | subnet                        |            3
- aws_elb                  | load_balancer                 |            0
- aws_elb                  | target_group                  |            0
- aws_elb                  | listener                      |            0
- aws_elb                  | load_balancer_security_groups |            0
- aws_ecs_fargate          | cluster                       |            0
- aws_ecs_fargate          | service                       |            0
- aws_ecs_fargate          | task_definition               |            0
- aws_ecs_fargate          | container_definition          |            0
- aws_ecs_fargate          | service_security_groups       |            0
-(17 rows)
+aws_cloudwatch | log_group | 0 aws_ecr | public_repository | 0 aws_ecr | repository | 1 aws_ecr | repository_policy | 0 aws_security_group | security_group | 2 aws_security_group | security_group_rule | 0 aws_vpc | vpc | 1 aws_vpc | subnet | 3 aws_elb | load_balancer | 0 aws_elb | target_group | 0 aws_elb | listener | 0 aws_elb | load_balancer_security_groups | 0 aws_ecs_fargate |
```

Comment:
I think there's a problem with line breaks here.

Reply:
you're right, i'll disable prettier for this